docs/development/npm-package-development.md

```diff
@@ -0,0 +1,168 @@
+# Npm Package Development
+
+## Recommended Components
+
+N/A
+
+## Guidance
+
+### Package Creation
+
+When starting the creation of a new package, it is recommended to use the `npm init` command instead of hand coding, to quickly and accuretly create a new package.json.  The results of accepting all the defaults, which can be done easily with this command `npm init -y`, is below:
```

Comment:
```suggestion
When starting the creation of a new package, it is recommended to use the `npm init` command instead of hand coding, to quickly and accurately create a new package.json.  The results of accepting all the defaults, which can be done easily with this command `npm init -y`, is below:
```